ui-v2/app/services/repository/token.js

```diff
@@ -6,6 +6,7 @@ import { Promise } from 'rsvp';
 import statusFactory from 'consul-ui/utils/acls-status';
 const status = statusFactory(Promise);
 const MODEL_NAME = 'token';
+const UNKNOWN_METHOD_ERROR = "rpc error making call: rpc: can't find method";
```

Comment:
If you wanted to be slightly more specific about it:
```
rpc error making call: rpc: can't find method ACL
```
would catch just ACL missing method errors (likely due to aclv2 upgrade)